README.md

```diff
@@ -19,9 +19,52 @@ For an example of this orb in use, see the GitHub repo [trussworks-atlantis-ecs-
 
 ## Developer Setup
 
-Install dependencies
+Install dependencies:
 
 ```sh
 brew install circleci pre-commit
 pre-commit install --install-hooks
 ```
+
+## Testing Changes
+
+After making changes to `orb.yml` and assuming you have the right credentials,
+you should be able to validate your changes:
+
+```sh
+circleci orb validate orb.yml
+```
+
+Once the validation is crystal, we can then publish a dev version to test with:
+
+```sh
+circleci orb publish orb.yml trussworks/orb-ecr-image-scan-findings@dev:first
+```
+
+Use this dev version to test in
+[trussworks-atlantis-ecs-image](https://github.com/trussworks/trussworks-atlantis-ecs-image)
+by creating a new branch and replacing the production version with your dev version.
+Commit and push this change. Confirm the CircleCI build was successful.
+
+If the build wasn't successful, edit the necessary changes to `orb.yml` and publish
```

Comment:
If I remember from pairing with you on this... there might be some quirks to just rerunning the workflow? Am I misremembering?

Reply:
True. It happened once and I still don't know what caused it. Except for that, rerunning worked.